File: alexandria/ui/src/io/intino/alexandria/ui/displays/components/SortBox.java
package io.intino.alexandria.ui.displays.components;

import io.intino.alexandria.core.Box;
import io.intino.alexandria.ui.displays.notifiers.SortBoxNotifier;

public class SortBox<DN extends SortBoxNotifier, B extends Box> extends AbstractSortBox<B> {

    public SortBox(B box) {
        super(box);
    }

}

File: alexandria/ui-elements/src/displays/components/SortBox.js
import React from "react";
import { withStyles } from '@material-ui/core/styles';
import AbstractSortBox from "../../../gen/displays/components/AbstractSortBox";
import SortBoxNotifier from "../../../gen/displays/notifiers/SortBoxNotifier";
import SortBoxRequester from "../../../gen/displays/requesters/SortBoxRequester";

const styles = theme => ({});

class SortBox extends AbstractSortBox {

	constructor(props) {
		super(props);
		this.notifier = new SortBoxNotifier(this);
		this.requester = new SortBoxRequester(this);
	};


}

export default withStyles(styles, { withTheme: true })(SortBox);